### PR TITLE
Jan elia 24/issue34

### PR DESCRIPTION
**Pull request by jan-elia-24**
Adding a pic to the header and fixind the dim & navbar & search 
kino

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,68 +1,67 @@
...
+  <meta charset="utf-8" />
+  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+  <title>Kino</title>
+  <meta name="viewport" content="width=device-width, initial-scale=1" />
+  <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+  <script type="module" src="main.js" defer></script>
+  <script src="https://kit.fontawesome.com/8357c99e29.js" crossorigin="anonymous"></script>
+  <!--The script above needs to be added on every singel html file that contains moviecard-->
+</head>
+
```
Comment (marcusmoller97) [inline index.html:14]: The main tag from index.html have dissapeared and a lot of section that should be present have also disappeared.

**Review by marcusmoller97 — CHANGES_REQUESTED** [review]
The header looks good.
There are some code that have disappeared from index.html that needs fixing.

**Review by marcusmoller97 — APPROVED** [review]
Everything seems to be in order now. Good work! 👍

**Review by kikoDevv — APPROVED** [review]
The design looks good! Nicely done!

**Review by itsJoel59 — APPROVED** [review]
Looks okay now, index.html seems fixed. Good job!

**Comment by marcusmoller97** [comment]
The tatus file are now pushed to repo. Maybe it shouldn't be there.

**Pull request merged**